libs/common/src/autofill/utils.ts

```diff
@@ -53,7 +51,7 @@ export function normalizeExpiryYearFormat(yearInput: string | number): Year | nu
 
 /**
  * Takes a cipher card view and returns "true" if the month and year affirmativey indicate
- * the card is expired.
+ * the card is expired. Uncertain cases return "false".
```

Comment:
⛏️ I usually use ticks to indicate literal values instead of strings to avoid confusion.
```suggestion
 * the card is expired. Uncertain cases return `false`.
```